library/kong

```diff
@@ -1,12 +1,23 @@
 Maintainers: Kong Docker Maintainers <support@konghq.com> (@thekonginc)
 GitRepo: https://github.com/Kong/docker-kong.git
 
-Tags: 2.8.0-alpine, 2.8.0, 2.8, alpine, latest
-GitCommit: c32a80c3dd332d096d1a6461d2816ead344d43c5
-GitFetch: refs/tags/2.8.0
-Directory: alpine
+Tags: 2.8.0-20220321-alpine, 2.8.0-alpine, 2.8-alpine, alpine, 2.8.0, 2.8, latest
```

Comment:
The calver tag is so we can have a mix of mutable (rebuild when a new base image is available) vs immutable.

I think this is OK as I see prior art examples though I couldn't find any official docs for or against this practice

Reply:
We don't actually support any "immutable" tags, so if Alpine is updated, `2.8.0-20220321-alpine` will be rebuilt.  What we suggest for users who require actual immutable image references is to use pull-by-digest instead, which is designed for cryptographic bit-for-bit reproducibility (ie, `docker pull kong:2.8@sha256:ec7199734e3af2d21dfcf566352b4cdf5c939ffcdf9a1cd0ad4f3ec6aac914b2` instead of just `kong:2.8`).